mizar/arktos/arktos_service.py

```diff
@@ -52,17 +55,26 @@ def CreatePod(self, request, context):
         param.body['status']['phase'] = request.phase
         param.body['metadata']['tenant'] = request.tenant
         param.extra = {}
+        ep_name = request.name+"-default-{}-eth0".format(request.tenant)
+        if ep_opr.store.get_ep(ep_name):
+            logger.info("EP {} already exists! Return OK".format(ep_name))
+            return ReturnCode(
```

Comment:
I think pod create and update are sharing same code. So if user wants to change pod by purpose, such as change its priority, seems to me the change will be unexpectedly ignored.

Reply:
Fixed.